### DISCUSSION TITLE
Can someone help me implement node-rate-limiter for acebase

### DISCUSSION THREAD

**Discussion by AAAATTIEH** [animir/node-rate-limiter-flexible#236] [Q&A] [answered]
Acebase is a database system similar to firebase but local. Any suggestions or code implementations on how to implement node-rate-limiter in it would be appreciated

**Comment by animir** [comment, marked as answer]
Hi @AAAATTIEH. I can help you with that.
You can start from [Contribution section](https://github.com/animir/node-rate-limiter-flexible#contribution) in Readme.
Then you can get `RateLimiterMongo`, `RateLimiterRedis` or any other limiter as example. All limiters are quite similar with some database specifics. After you finish `RateLimiterAcebase` class, you can copy Redis tests from [this file](https://github.com/animir/node-rate-limiter-flexible/blob/master/test/RateLimiterRedis.ioredis.test.js) and refactor them for `acebase` client instantiation.
If you have more questions, you may ask here or better create the new issue.
